specification/src/main/asciidoc/platform/PlatformOverview.adoc

```diff
@@ -1073,3 +1073,7 @@ SE 8 and to extend the range of validated objects. While the focus of
 CDI in this release is to extend its scope beyond Java EE with the
 introduction of a bootstrapping API, CDI also includes enhancements for
 event processing and alignment on Java SE 8 features.
+
+=== Changes in Jakarta EE 8
+
+Jakarta EE 8 is the migration of Java EE 8 from the JCP to the Eclipse Foundation.
```

Comment:
This obviously needs some more text.

Reply:
Actually, there shouldn't be much more.  We can talk about the required changes for the Spec Name changes and Acronym usage, but that's about all we're doing for Jakarta EE 8.